### DISCUSSION TITLE
Check Dogbag and Defi etc

### DISCUSSION THREAD

**Discussion by osmvermesser** [streetcomplete/StreetComplete#5224] [Ideas]
Hi!

i would prefer following Objekts

Dogbag (vending=excrement_bags)
* Exists
* bin
* Fee

Defi (emergency=defilibrator)
* exists
* Location

Postbox
* exists
* ref
* collection Times

Telefon (Germany Most will remove)
* exists

regards jan

**Comment by osmvermesser** [comment]
... additional
Postbox
* exists
* collection Times
* ref

Telephone (Germany many will remove)
* exists

jan

**Comment by HolgerJeromin** [comment]
Please do not throw random ideas into a discussion without researching the current state.
This discussion is difficult to adress. Many things are already implemented others are unclear.
Discussions about defi: #1756
postbox, telephone existence quest is already there.
collection Times for postboxes, too.

I am too lazy for doing more research now.

**Comment by osmvermesser** [comment]
Hallo Holger,

ich wusste nicht besser, wo ich den Vorschlag ablegen sollte.

Jan
⁣-----------------------------------
Jan's blickwinkel
blog.tappenbeck.net

Mein Verein für die Spur Z
http://www.efs-badschwartau.de

OpenStreetMap (OSM) - das FREIE Kartenprojekt
http://www.openstreetmap.de​

31.08.2023 08:03:25 Holger Jeromin ***@***.***>:
